README.md

```diff
@@ -9,7 +9,7 @@ specs conform to GoCardless's house style.
 To use `gc_ruboconfig`, simply add it to your Gemfile:
 
 ```ruby
-gem 'gc_ruboconfig', '~> 2.3.4'
+gem 'gc_ruboconfig', '~> 2.3.5'
```

Comment:
Shall we just get rid of the version bound here? People will know to add it if they want to, and it means we don't need to constantly update it.

Reply:
Sure!